test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1216,6 +1221,185 @@
+	<test id="Test 62-d: Check whether the classes are loaded correctly and work properly after the implementation has been modified" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $EXPORTS$ $currentMode$ -Xtrace:print={j9shr.2259,j9bcu.270} $CP_UNSAFE$ $PROGRAM_UNSAFE$</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">test done!</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">Another Test</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">World</output>		
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 0 for class name 'org/openj9/test/classtests/AnonClassAndUnsafeClassTest'</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 0 for class name 'org/openj9/test/classtests/AnonClassAndUnsafeClassTest/(00000000|0000000000000000)'</output>
+
+		<output type="failure" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 1 for class name 'org/openj9/test/classtests/AnonClassAndUnsafeClassTest'</output>
+		<output type="failure" caseSensitive="yes" regex="yes" javaUtilPattern="yes">compareROMClassForEquality returns 1 for class name 'org/openj9/test/classtests/AnonClassAndUnsafeClassTest/(00000000|0000000000000000)'</output>	
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 62 clean-up" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,destroy</command>
+		<output type="success" caseSensitive="yes" regex="no">Cache does not exist</output>
+		<output type="success" caseSensitive="yes" regex="no">has been destroyed</output>
+		<output type="success" caseSensitive="yes" regex="no">is destroyed</output>
+		
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 63-a: Only store the Anonymous Class in the cache" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$ -XX:-ShareUnsafeClasses $EXPORTS$ $CP_HANOI$ $PROGRAM_UNSAFE$</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">test done!</output>
```

Comment:
Might as well check for the unsafe class output here as well. Same for the anonymous class below.